client/my-sites/site-settings/form-general.jsx

```diff
@@ -500,7 +500,7 @@ export class SiteSettingsFormGeneral extends Component {
 					isSaving={ isSavingSettings }
 					onButtonClick={ handleSubmitForm }
 					showButton
-					title={ translate( 'Privacy' ) }
+					title={ translate( 'Privacy', { context: 'Privacy Settings header' } ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **143** times:
`translate( 'Privacy' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/add-context-to-privacy-settings-header)

Reply:
That's ok and the purpose of this PR. We'll copy all other translations with our Fuzzy Bot.